version.bzl

```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "1.0.9"
+DF_VERSION = "1.0.10"
```

Comment:
no need for this now.

Reply:
Isn't there? If I want to require this in `dataform-co` do I not refer to the version number? I guess I don't really understand the release process for this.